services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -308,5 +308,13 @@ export const generatePCCMTemplate = (originalReportTemplate: any) => {
     }
   }
 
+  // make additional form modifications as necessary
+  makePCCMTemplateModifications(reportTemplate);
+
   return reportTemplate;
 };
+
+const makePCCMTemplateModifications = (reportTemplate: ReportJson) => {
+  // Find Question C1.I.3 Program type in Section C.I and disable it
+  reportTemplate.routes[2].children![0].form!.fields[3].props!.disabled = true;
```

Comment:
at @benmartin-coforma 's suggestion I changed this to attempt to modify the intended field directly. this is more likely to cause issues if the template ever changes, which will bring this to our attention to fix sooner